### PR TITLE
Tel Aviv-North Control added

### PR DESCRIPTION
**Pull request by NelisV**
+ Northern split added 'Pluto'
+ Herzlia location changed

### CODE REVIEW

**Comment by NelisV** [comment]
also fixes #259

File: VATSpy.dat [hunk 1]
```diff
@@ -18009,7 +18009,8 @@ LJLA|Ljubljana||LJLA
 LKAA|Praha||LKAA
 LLLL|Tel Aviv|LLTA|LLLL
 LLLL|Tel Aviv|SHOLET|LLLL
-LLSC|Tel Aviv South Control||LLSC
+LLSC|Tel Aviv South||LLSC
+LLPT|Tel Aviv North|PLUTO|LLPT
 ; dummy codes for Israel CVFR centers
 LLPL|Pluto|PLUTO|LLLL
```
Comment (Adrian2k) [inline VATSpy.dat:18014]: 2 entries with Pluto as logon, should only be 1

**Review by Adrian2k — CHANGES_REQUESTED** [review]

File: VATSpy.dat (hunk 1 above)
Comment (NelisV) [reply VATSpy.dat:18014]: @prryplatypus can you check with the guy?

Comment (prryplatypus) [reply VATSpy.dat:18014]: From the request that we received it does look like this change is correct. I'm going to merge it in and if not we can fix it in a later version.

**Review by prryplatypus — APPROVED** [review]

**Pull request merged**